src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -71,12 +74,24 @@ let nextContentId: number = 0;
 })
 export class SkyRepeaterItemComponent implements OnDestroy, OnInit, AfterViewInit {
 
+  /**
+   * Specifies configuration options for the buttons to display on an inline form
+   * within the repeater. This property accepts a `SkyInlineFormConfig` object.
+   */
   @Input()
   public inlineFormConfig: SkyInlineFormConfig;
 
+  /**
+   * Specifies a template to use to instantiate an inline form within the repeater.
```

Comment:
Again, based on our styles in phase 2 docs, we should remove the `This property accepts...` sentence. But the link seems valuable here. I wonder if it would work to we reworded this description as: `Specifies [an Angular TemplateRef class](https://angular.io/api/core/TemplateRef). to use as a template to instantiate an inline form within the repeater.`

Also, should `TemplateRef` be formatted as a code snippet? I think so.

Reply:
Great suggestion. Done.